src/de/jost_net/JVerein/io/Rechnungsausgabe.java

```diff
@@ -102,6 +104,19 @@ public Rechnungsausgabe(MitgliedskontoControl control) throws IOException
       mks = getRechnungsempfaenger(mk);
     }
     aufbereitung(formular);
+    try
+    {
+      // Write updated form to DB
+      formular.store();
+      // Update all linked forms
+      formular.setZaehlerToFormLink(formular.getZaehler());
+    }
+    catch (Exception e)
+    {
+      String fehler = "Formularfeld kann nicht gespeichert werden. Siehe system log";
+      Logger.error(fehler, e);
+      throw new RemoteException(fehler);
```

Comment:
Wenn du möchtest, dass die sprechende Fehlermeldung dem User oben in dem Hinweisbanner angezeigt wird, kannst du hier stattdessen eine "ApplicationException" werfen. Wobei man hier aber testen muss, ob die ApplicationException dann auch weit genug hoch bis Jameica gelassen wird oder zwischendurch nochmal gefangen wird.